### PR TITLE
Update data-only-hotspots.mdx

### PR DESCRIPTION
**Pull request by dtischler**
Add a note that paying this Assert fee is different than purchasing data credits.

### CODE REVIEW

**Comment by shawaj** [comment]
@dtischler they technically are data credits, but you can't use them from the Console you can only use them from the CLI wallet. Or having HNT in your wallet and using the "implicit burn" 

Reason for this, I think, is that the console is essentially one wallet that Helium themselves control and the data credits your purchase there are just apportioned to your account on Console. If that makes sense

File: docs/mine-hnt/data-only-hotspots.mdx [hunk 1]
```diff
@@ -43,6 +43,10 @@ The fee schedule is as follows:
 * **add data only hotspot**: 1000000 DC ($10)
 * **assert location**: 500000 DC ($5)
 
+:::info
+Note:  These fees are not the same thing as Data Credits, and purchasing Data Credits in Console is not a method of paying this fee.  Instead, this fee needs to be paid by transferring funds from a wallet such as Binance or similar.
```
Comment (PharkMillups) [inline docs/mine-hnt/data-only-hotspots.mdx]:
```suggestion
Data Credits in your Helium Console organization cannot be used to pay blockchain onboarding and assert location fees for Hotspots. To onboard a Hotspot, you'll need to burn HNT for Data Credits to the wallet that owns the Hotspot.
```

**Comment by PharkMillups** [comment]
@dtischler Thanks for the contrib! I made a few changes to the text to clarify what you mean (because it is useful). Hopefully this reads ok to you.

**Comment by dtischler** [comment]
Looks perfect, go for it @PharkMillups !

**Review by PharkMillups — APPROVED** [review]

**Comment by PharkMillups** [comment]
@dtischler Thanks for the contrib! Ping me in the docs channel on discord so I can give you the docs contributor role.

**Pull request merged**